src/app/routes/cpsAsset/getInitialData/index.js

```diff
@@ -137,7 +139,7 @@ export default async ({
     const response = {
       status,
       pageData: {
-        ...(await transformJson(article, pathname, toggles)),
+        ...(isCaf ? article : await transformJson(article, pathname, toggles)),
```

Comment:
I guess this is the cruical point where we could delete everything below `transformJson` in the call stack if made this permanent 🥳 ?

Reply:
Yep thats right! The entire `transformJson` and all those Optimo processing bits can be removed.